.github/workflows/test-and-release.yml

```diff
@@ -25,7 +25,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x]
+        node-version: [12.x, 15.x]
```

Comment:
why you did that change? Should be unneeded here

Reply:
You are right. Just go over it.

Reply:
could you please remove it again?  :-)